django/utils/log.py

```diff
@@ -189,3 +192,43 @@ def format(self, record):
 
         record.msg = msg
         return super(ServerFormatter, self).format(record)
+
+
+def log_response(message, *args, **kwargs):
+    """Logs errors based on response status.
+
+    Log 5xx responses in error level and 4xx in warning level (unless level
+    is explicitly given as a keyword argument). The Response status_code and
+    the entire request are passed to logger extra parameter. This function
+    also avoid duplicated logs.
+
+    """
+
+    for argname in ('request', 'response'):
+        if argname not in kwargs:
+            raise TypeError("'log_response' call missing the required keyword argument: '{}'".format(argname))
+
+    response = kwargs.pop('response')
+    request = kwargs.pop('request')
+    logger = kwargs.pop('logger', request_logger)
+    level = kwargs.pop('level', None)
```

Comment:
What if we make `level` default to "info" or "debug" instead? That way if you call `log_response` on a non-error response you'd just get an info/debug log, instead of getting a confusing "module logger has no attribute None" error (or whatever the wording actually would be)